test-files/cases/hoot-js/line-generic-1/Expected.osm

```diff
@@ -124,8 +103,31 @@
         <nd ref="-27620"/>
         <nd ref="-27621"/>
         <nd ref="-27622"/>
+        <tag k="highway" v="road"/>
         <tag k="note" v="0b 2;0"/>
+        <tag k="error:circular" v="15"/>
+    </way>
+    <way visible="true" id="-27641" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-27565"/>
+        <nd ref="-27566"/>
+        <nd ref="-27567"/>
+        <nd ref="-27568"/>
+        <nd ref="-27569"/>
+        <nd ref="-27570"/>
+        <nd ref="-27571"/>
+        <nd ref="-27572"/>
+        <nd ref="-27573"/>
+        <nd ref="-27574"/>
+        <nd ref="-27575"/>
+        <nd ref="-27576"/>
+        <nd ref="-27577"/>
+        <nd ref="-27578"/>
+        <nd ref="-27579"/>
+        <nd ref="-27580"/>
+        <nd ref="-27581"/>
+        <nd ref="-27582"/>
```

Comment:
Some outputs are mostly just node and way ordering by different IDs.  No problem at all.